flink-yarn/src/main/java/org/apache/flink/yarn/YarnResourceManager.java

```diff
@@ -419,6 +398,36 @@ public void onContainersAllocated(List<Container> containers) {
 		});
 	}
 
+	private void startTaskExecutorInContainer(Container requiredContainer) {
```

Comment:
wondering whether the `required` prefix makes sense in the context of this method; I'd tend to say no.

Reply:
True, will change it.